lib/config/presets/internal/config.ts

```diff
@@ -15,6 +15,16 @@ export const presets: Record<string, Preset> = {
       'workarounds:all',
     ],
   },
+  'best-practices': {
+    configMigration: true,
+    description: 'Preset with best practices from the Renovate maintainers.',
+    extends: [
+      'config:base',
```

Comment:
@viceice This should be `'config:recommended'`, or? I noticed that in documentation. Not sure what should be updated. Only this file and docs will be generate automatically?

Reply:
The `config:best-practices` preset still uses `config:base` in the current `main` branch:

https://github.com/renovatebot/renovate/blob/c89eeb89dff3ee5a62b71c936bd447e579512e27/lib/config/presets/internal/config.ts#L1-L15

I think this was caused because we merged the `config:best-practices` preset to `main` before we merged the `config:base -> `config:recommended` PR. The `config:base` rename PR didn't fix up this file.